.npmignore

```diff
@@ -54,6 +53,8 @@ results.html
 *.lcov
 
 # generated files needed for publish
+dist/*
+dist/report/*
```

Comment:
this lets us ignore the other files in `dist/report/`, which I assume we want to do

```sh
npm notice 177.4kB dist/report/bundle.js                                                               
npm notice 165.6kB dist/report/psi.js                                                                  
npm notice 74.7kB  dist/report/standalone.js                                                           
npm notice 12.3kB  dist/report/treemap.js                                                              
npm notice 188.3kB dist/report/viewer.js  
```